### PR TITLE
Don't call deprecated InstalledVersions::getRawData()

### PR DESCRIPTION
**Pull request by derrabus**
Backport of Ocramius/ProxyManager#700

### CODE REVIEW

File: src/ProxyManager/Generator/Util/IdentifierSuffixer.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
 use Composer\InstalledVersions;
 
 use function class_exists;
+use function is_callable;
```
Comment (nicolas-grekas) [inline src/ProxyManager/Generator/Util/IdentifierSuffixer.php]: Let's use method exists

Comment (derrabus) [reply src/ProxyManager/Generator/Util/IdentifierSuffixer.php]: Done.

**Comment by nicolas-grekas** [comment]
Thank you!

**Comment by alexander-schranz** [comment]
@derrabus  Thank you for this, it works also great when having older composer versions installed 👍

**Pull request merged**